web/src/app/administration/components/classification-overview/classification-overview.component.ts

```diff
@@ -36,6 +37,7 @@ export class ClassificationOverviewComponent implements OnInit, OnDestroy {
           if (this.routerParams.id) {
             this.showDetail = true;
             this.store.dispatch(new SelectClassification(this.routerParams.id));
+            setTimeout(() => this.store.dispatch(new GetClassifications()), 0);
```

Comment:
what's the reason for this timeout? is this really necessary? If yes, please document the reason

Reply:
I changed this to a subscription. We have to wait for the first action to finish in order to get the classifications which belong to the new classification type.